judge/models/tests/test_problem.py

```diff
@@ -0,0 +1,183 @@
+from django.test import TestCase
+
+from judge.models import Language, LanguageLimit, Problem
+from judge.models.tests.util import CommonDataMixin, create_problem, create_problem_type
+
+
+class ProblemTestCase(CommonDataMixin, TestCase):
```

Comment:
From codecov analysis, test changing of problem `code` and that when `case_points == case_total`, `result_class_from_code` returns `AC`.

Reply:
Done the testing `result_class_from_code` returning `AC`. However, I think that testing problem code changing ought to be done alongside `ProblemData` testing, since there is no way to really test this properly currently (unless we just test that `save()` works?)

Reply:
Yep, testing this alongside `ProblemData` makes sense.